code/modules/projectiles/boxes_magazines/internal/_cylinder.dm

```diff
@@ -13,9 +13,14 @@
 			update_appearance()
 			return
 
-/obj/item/ammo_box/magazine/internal/cylinder/get_round()
-	rotate()
-	return stored_ammo[1]
+/obj/item/ammo_box/magazine/internal/cylinder/get_round(keep = FALSE)
```

Comment:
this seems a bit clunky, ideally we'd have two procs, one that hands back the top round and one hands back an unowned top round